pages/common/x11docker.md

```diff
@@ -0,0 +1,25 @@
+# x11docker
+
+> Run GUI applications and desktops in docker giving focus on security.
+> See also `xephyr`.
+> More information: <https://github.com/mviereck/x11docker>.
+
+- Launch VLC in a container:
+
+`x11docker --pulseaudio --share={{$HOME/Videos}} {{jess/vlc}}`
```

Comment:
What does `jess` mean?

Reply:
That's the container image name. https://hub.docker.com/r/jess/vlc.  @<!---->jessfraz is famous for her Docker work ;)

Reply:
As someone, who dosn't know much about docker, the examples descriptions are a bit too generic / the examples are a bit too specific. But again, if you know docker you probably also understand all the examples, so I think it's fine.

Reply:
Well, this example is about lauching VLC 🤔